### DISCUSSION TITLE
help with proxxies

### DISCUSSION THREAD

**Discussion by pl4udible** [3kh0/ext-remover#489] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
anyone kow any proxxies

**Comment by fnfgo** [comment, marked as answer]
> anyone kow any proxxies

what does kow mean i dont think thats a word.
you shouldnt be looking for proxies when youre in first grade.

**Reply by fnfgo** [reply]
> > anyone kow any proxxies
> 
> what does kow mean i dont think thats a word. you shouldnt be looking for proxies when youre in first grade.

but here are some proxy links 
([](https://www.youtube.com/watch?v=MEV-WgaUsBk&t=46s))
and some more
[](https://www.youtube.com/watch?v=M5V_IXMewl4)

**Reply by fnfgo** [reply]
> > > anyone kow any proxxies
> > 
> > 
> > what does kow mean i dont think thats a word. you shouldnt be looking for proxies when youre in first grade.
> 
> but here are some proxy links () and some more
https://www.youtube.com/watch?v=M5V_IXMewl4
https://www.youtube.com/watch?v=MEV-WgaUsBk&t=46s

**Reply by bartholemuew** [reply]
bro its just a spelling mistake
